src/operation/decision_model_store.jl

```diff
@@ -122,9 +140,11 @@ function write_optimizer_stats!(
     stats::OptimizerStats,
     index::DecisionModelIndexType,
 )
-    if index in keys(store.optimizer_stats)
-        @warn "Overwriting optimizer stats"
-    end
+    # TODO: This check is incompatible with test calls to psi_checksolve_test
```

Comment:
It would be nice to keep this check. Would need to workaround the test behavior.